llama_stack/providers/remote/vector_io/chroma/chroma.py

```diff
@@ -57,12 +57,16 @@ def __init__(self, client: ChromaClientType, collection, kvstore: KVStore | None
         self.collection = collection
         self.kvstore = kvstore
 
+    async def initialize(self):
+        # Chroma does not require explicit initialization, this is just a helper for unit tests
```

Comment:
delete this comment, not clear what "helper for unit tests" means even

Reply:
removed 👍